examples/optioncontainer/optioncontainer/app.py

```diff
@@ -0,0 +1,110 @@
+import toga
+from toga.style import Pack
+from toga.constants import COLUMN, ROW
+
+
+class ExampleOptionContainerApp(toga.App):
+
+    def _create_options(self):
+        label_box1 = toga.Label('This is Box 1 ', style=Pack(padding=10))
+        label_box2 = toga.Label('This is Box 2 ', style=Pack(padding=10))
+        label_box3 = toga.Label('This is Box 3 ', style=Pack(padding=10))
+
+        box1 = toga.Box(children=[label_box1])
+        box2 = toga.Box(children=[label_box2])
+        box3 = toga.Box(children=[label_box3])
+
+        self.optioncontainer.add('Option1', box1)
+        self.optioncontainer.add('Option2', box2)
+        self.optioncontainer.add('Option3', box3)
+        self._refresh_select()
+
+    def _remove_all(self):
+        while len(self.optioncontainer.content) > 0:
+            self.optioncontainer.remove(0)
+
+    def _refresh_select(self):
+        items = []
+        for i in range(len(self.optioncontainer.content)):
+            items.append(str(i))
+        self.select_option.items = items
+
+    def on_add_option(self, button):
+        self.optioncontainer.add('Option', toga.Box())
+        self._refresh_select()
+
+    def on_enable(self, button):
+        index = int(self.select_option.value)
+        self.optioncontainer.content[index].enabled = not self.optioncontainer.content[index].enabled
+
+    def on_change_title(self, button):
+        index = int(self.select_option.value)
+        self.optioncontainer.content[index].label = self.input_change_title.value
+
+    def on_remove(self, button):
+        index = int(self.select_option.value)
+        del self.optioncontainer.content[index]
+        self._refresh_select()
+
+    def on_reset_optioncontainer(self, button):
+        self._remove_all()
```

Comment:
This fails if you have any disabled tab items.

Reply:
I removed this reset feature because it was imposible with the new delete restrictions applied in the last commit